apps/web-giddh/src/app/reports/components/purchase-register-table-component/purchase.register.table.component.html

```diff
@@ -2,8 +2,30 @@
     (localeData)="localeData = $event" (commonLocaleData)="commonLocaleData = $event">
     <table class="table basic giddh-table on-mobile-view">
         <thead>
+                        <tr class="register-table-th">
```

Comment:
Please check the formatting of this section, appears to be more than 4 spaces in the diff

Reply:
done